pkg/k8s/graceful_shutdown.go

```diff
@@ -0,0 +1,74 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	api "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+)
+
+var (
+	stopCH = make(chan struct{}, 0)
```

Comment:
1. 0 is not needed
2. `stopCH` can be local var instead of global

Reply:
3d14adf9